vocode/streaming/utils/worker.py

```diff
@@ -33,9 +63,6 @@ def start(self) -> asyncio.Task:
     def consume_nonblocking(self, item: WorkerInputType):
         self.input_queue.put_nowait(item)
 
-    def produce_nonblocking(self, item):
```

Comment:
i think i checked all of the cases, and it would hopefully be caught by mypy (but we know that this isn't always reliable)

workers no longer have output queues and can have multiple consumers, which can be referenced however the worker wants it to be referenced, so the superclass can't implement `produce_nonblocking`